SoftLayer/managers/hardware.py

```diff
@@ -80,6 +94,10 @@ def cancel_metal(self, hardware_id, immediate=False):
         :param bool immediate: If true, the bare metal instance will be
                                cancelled immediately. Otherwise, it will be
                                scheduled to cancel on the anniversary date.
+        Example::
+
+            result = mgr.cancel_metal(hardware_id=1234)
```

Comment:
cancel_metal() is being removed in https://github.com/softlayer/softlayer-python/pull/524 because it doesn't seem needed any longer as cancel_hardware() can serve the same purpose now.